File: src/kyu_8/quadrants/python/quadrants_test.py
import codewars_test as test
from solution import quadrant

@test.describe("Basic Tests")
def basic_tests():

    @test.it("Basic Tests")
    def basic_tests():

        tests = [[1, 2, 1], [3, 5, 1], [-10, 100, 2], [-1, -9, 3], [19, -56, 4], [1, 1, 1], [-60, -12, 3]]

        for x, y, expected in tests:
            test.assert_equals(quadrant(x, y), expected, f"{x = }, {y = }")

File: src/kyu_8/quadrants/csharp/quadrants_test.cs
namespace Solution
{
  using NUnit.Framework;
  using System;

  [TestFixture]
  public class SolutionTest
  {
    static void Act(int x, int y, int expected)
      => Assert.AreEqual(expected, Kata.Quadrant(x, y), $"Quadrant( x = {x}, y = {y} )");

    [TestCase(1, 2, 1)]
    [TestCase(3, 5, 1)]
    [TestCase(-10, 100, 2)]
    [TestCase(-1, -9, 3)]
    [TestCase(19, -56, 4)]
    [TestCase(1, 1, 1)]
    [TestCase(-60, -12, 3)]
    public void FixedTests(int x, int y, int expected) => Act(x, y, expected);
  }
}